test/unit/offset.js

```diff
@@ -462,9 +462,9 @@ testIframe("offset/body", "body", function( $ ) {
 test("chaining", function() {
 	expect(3);
 	var coords = { "top":  1, "left":  1 };
-	equal( jQuery("#absolute-1").offset(coords).selector, "#absolute-1", "offset(coords) returns jQuery object" );
-	equal( jQuery("#non-existent").offset(coords).selector, "#non-existent", "offset(coords) with empty jQuery set returns jQuery object" );
-	equal( jQuery("#absolute-1").offset(undefined).selector, "#absolute-1", "offset(undefined) returns jQuery object (#5571)" );
+	equal( jQuery("#absolute-1").offset(coords).jquery, jQuery.fn.jquery, "offset(coords) returns jQuery object" );
```

Comment:
Until I looked at the old PR, I had just planned to see if the `.selector` property was present but the change by @gibson042 seemed like a slightly better assertion. 

I guess `.version` was avoided since it's a pretty common name. You wouldn't think `someobject.fn.version` would be that ambiguous but it's also available via `someobject.prototype.version` so maybe that caused concern.